### PR TITLE
Ingested EAMT 2020

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

File: data/xml/2020.eamt.xml [hunk 1]
```diff
@@ -0,0 +1,728 @@
...
+% of elliptical utterances.</abstract>
+      <url hash="a50e8e6c">2020.eamt-1.30</url>
+    </paper>
+    <paper id="31">
+      <title>Bifixer and Bicleaner: two open-source tools to clean your parallel data</title>
+      <author><first>Gema</first><last>Ramírez-Sánchez</last></author>
+      <author><first>Jaume</first><last>Zaragoza-Bernabeu</last></author>
+      <author><first>Marta Ba
+
+textasciitilde</first><last>nón</last></author>
```
Comment (danielgildea) [inline data/xml/2020.eamt.xml:309]: there are several of these

File: data/xml/2020.eamt.xml [hunk 2]
```diff
@@ -0,0 +1,728 @@
...
+      <title>Ellipsis Translation for a Medical Speech to Speech Translation System</title>
+      <author><first>Jonathan</first><last>Mutal</last></author>
+      <author><first>Johanna</first><last>Gerlach</last></author>
+      <author><first>Pierrette</first><last>Bouillon</last></author>
+      <author><first>Hervé</first><last>Spechbach</last></author>
+      <pages>281–290</pages>
+      <abstract>In diagnostic interviews, elliptical utterances allow doctors to question patients in a more efficient and economical way. However, literal translation of such incomplete utterances is rarely possible without affecting communication. Previous studies have focused on automatic ellipsis detection and resolution, but only few specifically address the problem of automatic translation of ellipsis. In this work, we evaluate four different approaches to translate ellipsis in medical dialogues in the context of the speech to speech translation system BabelDr. We also investigate the impact of training data, using an under-sampling method and data with elliptical utterances in context. Results show that the best model is able to translate 88
+
+% of elliptical utterances.</abstract>
```
Comment (danielgildea) [inline data/xml/2020.eamt.xml:300]: blank line

File: data/xml/2020.eamt.xml (hunk 1 above)
Comment (mjpost) [reply data/xml/2020.eamt.xml:309]: Hmm. Here's the line in the bibtex:

```bibtex
author = "Gema Ram\'{\i}rez-S\'{a}nchez and Jaume Zaragoza-Bernabeu and Marta Ba\\textasciitilde {n}\'{o}n and Sergio Ortiz Rojas",
```

I wonder if this is an error in ACLPUB. It's not clear how to deal with it on the Anthology side, so I'm just going to correct it manually.

File: data/xml/2020.eamt.xml (hunk 2 above)
Comment (mjpost) [reply data/xml/2020.eamt.xml:300]: This was `...able to translate 88\\% of elliptical utterances.",` in the BibTeX, FYI. I'll just correct manually.

**Pull request merged**